docs/reference/migration/migrate_8_0/java.asciidoc

```diff
@@ -9,27 +9,36 @@
 
 // end::notable-breaking-changes[]
 
-[float]
-==== Changes to Fuzziness
-
+.Changes to fuzziness
+[%collapsible]
+====
+*Details* +
 To create `Fuzziness` instances, use the `fromString` and `fromEdits` method
 instead of the `build` method that used to accept both Strings and numeric
 values. Several fuzziness setters on query builders (e.g.
 MatchQueryBuilder#fuzziness) now accept only a `Fuzziness`instance instead of
-an Object. You should preferably use the available constants (e.g.
-Fuzziness.ONE, Fuzziness.AUTO) or build your own instance using the above
-mentioned factory methods.
+an Object.
 
 Fuzziness used to be lenient when it comes to parsing arbitrary numeric values
 while silently truncating them to one of the three allowed edit distances 0, 1
 or 2. This leniency is now removed and the class will throw errors when trying
 to construct an instance with another value (e.g. floats like 1.3 used to get
-accepted but truncated to 1). You should use one of the allowed values.
-
-
-[float]
-==== Changes to Repository
-
+accepted but truncated to 1).
+
+*Impact* +
+Use the available constants (e.g. Fuzziness.ONE, Fuzziness.AUTO) or build your
+own instance using the above mentioned factory methods. Use only allowed
+`Fuzziness` values.
```

Comment:
@cbuescher Do you mind confirming that this accurately reflects your original guidance?

This is mostly a formatting change. Mostly want to ensure I don't unintentionally lose any context here.

Feel free to ignore the other changes in this PR unless you'd like to review them.